virtcontainers/device/drivers/vfio.go

```diff
@@ -82,13 +93,16 @@ func (device *VFIODevice) Attach(devReceiver api.DeviceReceiver) error {
 		"device-group": device.DeviceInfo.HostPath,
 		"device-type":  "vfio-passthrough",
 	}).Info("Device group attached")
-
+	device.DeviceInfo.Hotplugged = true
 	return nil
 }
 
 // Detach is standard interface of api.Device, it's used to remove device from some
 // DeviceReceiver
 func (device *VFIODevice) Detach(devReceiver api.DeviceReceiver) error {
+	device.DevManager.Lock()
+	defer device.DevManager.Unlock()
+	device.DeviceInfo.Hotplugged = true
```

Comment:
Shouldnt this be false?

Reply:
Oh, right. A critical mistake. Will modify